docs/source/release/v3.12.0/framework.rst

```diff
@@ -65,6 +65,7 @@ Core Functionality
 - Added new classes ``ConfigObserver`` for listening for changes to any configuration property and ``ConfigPropertyObserver`` for listening to changes to an individual config property of interest.
 - Fixed the calculation of scattering length and scattering length squared for :py:obj:`Material <mantid.kernel.Material>`
 - Fixed the behaviour of ``UpdateInstrumentDefinitions.OnStartup`` in the :ref:`properties file <Properties File>`. It was not being used correctly for using the updated ``Facilities.xml`` file.
+- ``MultiFileProperty`` now accepts complex summation ranges for run numbers, such as ``111-113+115`` and ``111-115+123-132``.
```

Comment:
Question:
Is it potentially confusing to have `-` for defining a range but `+` combining ranges? Is this convention following any existing behavior?

Reply:
`+` and `-` deal with adding (merging) of data. The documentation of `MultipleFileProperty` states that 'Added Ranges and Stepped Added Ranges may be plussed however.' At least to me the behavior of adding 'added ranges' with `+` feels logical, and the use pattern will be common at here at ILL where users frequently merge several files into one workspace.

I'll see if I can improve the documentation. At least `MultipleFileProperty` could include examples of this kind, maybe `Load`, `LoadAndMerge` and some tooltip could be improved as well

Reply:
My point is: Is there a risk that having `-` for the range is interpreting as *subtracting* that run from the others? If so, would `:` work better?

Reply:
There is indeed that risk, but `-` is and has always been an 'Added Range', also according to `MultipleFileProperty` documentation. `:` is a range of individual files so it cannot be used for summing. I don't think `n:m+k:l` makes sense, and is explicitly prohibited in `MultipleFileProperty` documentation.

Reply:
Good point, I did not know that. I guess then we do not have a choice and just keep it as it is. :)